functorch/_src/eager_transforms.py

```diff
@@ -609,6 +605,11 @@ def assert_flat_tuple_of_tensors(elts, api, argname):
             f'{api}: Expected {argname} to be a non-empty tuple of Tensors.')
 
 
+def assert_non_empty_output(output, api):
+    if output == [None] or len(output) < 1:
+        raise RuntimeError(f'{api}: Expected non-empty output of f')
```

Comment:
nit: We can make this more descriptive: `"Expected f to be a function that has non-empty output (got output = {output})`